plugins/Terminal/js/components/seedprompt.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+const querystring = require('querystring')
+
+const WalletSeedPrompt = ({ showSeedPrompt, currentCommand, actions }) => {
+	componentDidUpdate: {
```

Comment:
Idea was to auto focus to the most prominent textbox automatically. For the wallet/seed modals those textboxes needed to be drawn first. Using `componentDidUpdate` allowed me to schedule the focus call to after the modals were visible. You mentioned event listeners might be a better way to handle this.